bciers/apps/reporting/src/app/bceidbusiness/industry_user/new-entrant-information/page.tsx

```diff
@@ -0,0 +1,14 @@
+import { Suspense } from "react";
+import Loading from "@bciers/components/loading/SkeletonForm";
+import NewEntrantInformation from "@reporting/src/app/components/additionalInformation/newEntrantInformation/NewEntrantInformation";
+interface PageProps {
+  params: { version_id: number };
+}
+
+export default async function Page({ params }: PageProps) {
+  return (
+    <Suspense fallback={<Loading />}>
+      <NewEntrantInformation versionId={params.version_id} />
+    </Suspense>
+  );
+}
```

Comment:
this can use the `defaultPageFactory`, you'll just need to make sure that the `NewEntrantInformation` component takes a `version_id` param instead of `versionId`